build/archives/BUILD.gn

```diff
@@ -74,6 +88,15 @@ if (build_engine_artifacts) {
         destination = "libtessellator$dll"
       },
     ]
+    if (host_os == "mac" && full_target_platform_name == "darwin-x64") {
```

Comment:
Do we need to sign arm64 artifacts?

Reply:
My reasoning is https://github.com/flutter/engine/pull/34112#discussion_r906425320. Maybe @christopherfujino could help decide if it is a better choice to have a tight constraint?

Reply:
we don't currently build arm artifacts; however, I don't think you should exclude arm here.